Source/Diagnostics/ComputeDiagFunctors/DivBFunctor.cpp

```diff
@@ -12,7 +12,7 @@ DivBFunctor::operator()(amrex::MultiFab& mf_dst, int dcomp) const
 {
     auto& warpx = WarpX::GetInstance();
     // Guard cell is set to 1 for generality. However, for a cell-centered
-    // output Multifab, mf_avg, the guard-cell data is not needed especially considering
+    // output Multifab, m_mf_output, the guard-cell data is not needed especially considering
```

Comment:
Shouldn't't this actually be `mf_dst`?

Reply:
That is right! Changed it now. Thank you!!